Assignment-1-6/pdf.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+require(__DIR__ . '/fpdf186/fpdf.php');
+
+session_start();
+
+$firstName = $_SESSION['firstName'];
+$lastName = $_SESSION['lastName'];
+$phone = $_SESSION['phone'];
+$email = $_SESSION['email'];
+
+// Session unset & destroy
+session_unset();
+session_destroy();
```

Comment:
This is a utility file for generating PDFs.
Why are we working with session here?

Reply:
Still not fixed.

Reply:
@bimandas1 This is the 3rd time I am having to mention that this is not fixed.